src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/ResourceServerConfiguration.kt

```diff
@@ -35,7 +35,7 @@ class ResourceServerConfiguration {
         ).forEach { authorize(it, permitAll) }
         authorize(anyRequest, authenticated)
       }
-      oauth2ResourceServer { jwt { jwtAuthenticationConverter = AuthAwareTokenConverter() } }
+      oauth2ResourceServer { jwt { jwtAuthenticationConverter = tokenConverter } }
```

Comment:
I'm not sure why we were new'ing the `AuthAwareTokenConverter` up here rather than injecting it, but seeing as the `AuthAwareTokenConverter` now has a constructor arg (the HMPPS Auth Client) we need to inject it rather than new it up.